sites/watertechonline.com/server/styles/index.scss

```diff
@@ -24,3 +24,20 @@ $theme-site-header-breakpoints: (
 
 @import "../../node_modules/@endeavor-business-media/package-shared/scss/core";
 @import "../../node_modules/@endeavor-business-media/package-common/scss/algolia";
+
+// Custom styling for global-thought-leaders website section
+
+/* stylelint-disable */
```

Comment:
@Alixers505  We shouldn't disable lint to get this done.  With the change from class to modifier in the section page override the correct version should look something like this

```SCSS
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper {
        height: auto;
        min-height: auto;
        margin: 0 auto;
      }
    }
  }
}
```

Reply:
The height and min-height settings are manually set within the component to display inline in the html. So it requires "!important" to override it. Important isn't allowed by the linter settings.

Reply:
@Alixers505  to that point those are really un needed.  The height is a calculated value based off the width.  All you should need to achieve this look is 
```SCSS
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper {
        margin: 0 auto;
      }
    }
  }
}
```

Reply:
*min-width, not min-height

Reply:
@B77Mills - that does solve it for desktop, yes, but not for mobile phones. You can see it drastically offset on older phones and slightly offset on newer phones:
<img width="968" alt="Screen Shot 2020-10-23 at 9 30 09 AM" src="https://user-images.githubusercontent.com/6343242/97009693-6ac7b680-1512-11eb-8d18-b0c00d0bdb50.png">
<img width="476" alt="Screen Shot 2020-10-23 at 9 31 05 AM" src="https://user-images.githubusercontent.com/6343242/97009796-8d59cf80-1512-11eb-8d2f-2eb19f071324.png">
<img width="454" alt="Screen Shot 2020-10-23 at 9 30 53 AM" src="https://user-images.githubusercontent.com/6343242/97009800-8e8afc80-1512-11eb-8d6c-26dc05711260.png">
<img width="469" alt="Screen Shot 2020-10-23 at 9 31 57 AM" src="https://user-images.githubusercontent.com/6343242/97009850-9f3b7280-1512-11eb-9b5b-35e083151fe2.png">

Reply:
If the important thing needs to stay we should still only disable lint on the single lines 
```scss
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper,
      &__image {
        min-width: auto !important; /* stylelint-disable-line */
        height: auto !important; /* stylelint-disable-line */
        margin: 0 auto;
      }
    }
  }
}
```

This with the other change to the file should work with a more narrowed down selector for the css override.